doc/Pacemaker_Explained/en-US/Ch-Resources.txt

```diff
@@ -519,7 +519,7 @@ but moderate. The minimum timeouts should never be below 10 seconds.
 Location to store the resource state in.
 </longdesc>
 <shortdesc lang="en">State file</shortdesc>
-<content type="string" default="/var/run//Dummy-{OCF_RESOURCE_INSTANCE}.state" />
+<content type="string" default="/var/run//Dummy-default.state" />
```

Comment:
Another consideration is that UIs that know the resource ID could replace the placeholder text intelligently (see #937 ). What about:

<content type="string" default="/var/run/Dummy-${OCF_RESOURCE_INSTANCE:-RESOURCE_ID}.state" />

If the caller knows the resource name and provides it as OCF_RESOURCE_INSTANCE, it will get the "really correct" output, otherwise it will just show the string RESOURCE_ID, which is more likely to be meaningful to a sysadmin configuring the cluster than code would be.